### ISSUE TITLE
Port SRB1_1_8 changes to Linux x86

### ISSUE THREAD

**Issue by mbjones** [NCEAS/metacat#87]
---

Author Name: **Chris Jones** (Chris Jones)
Original Redmine Issue: 152, https://projects.ecoinformatics.org/ecoinfo/issues/152
Original Date: 2000-09-23
Original Assignee: Owen Eddins

---

Port SRB1_1_8 changes to Linux x86.  SRB1_1_7 has been ported to run on Linux,
but in parallel to changes made to SRB1_1_8.  Linux specific code changes need
to be moved to the 1_1_8 code.

**Comment by mbjones** [comment]
---

Original Redmine Comment
Author Name: **Matt Jones** (Matt Jones)
Original Date: 2000-10-14T22:41:51Z

---

Reassigned to Owen because he has main responsibility for the SRB work.

**Comment by mbjones** [comment]
---

Original Redmine Comment
Author Name: **Matt Jones** (Matt Jones)
Original Date: 2001-04-12T16:48:17Z

---

Lowered priority because SRB no longer in immediate plans for deployment.

**Comment by mbjones** [comment]
---

Original Redmine Comment
Author Name: **Matt Jones** (Matt Jones)
Original Date: 2001-05-11T08:45:46Z

---

Deferred indefinitely because SRB deployment is in question.  Will reopen later 
if we decide to deploy the SRB.

**Comment by mbjones** [comment]
---

Original Redmine Comment
Author Name: **Matt Jones** (Matt Jones)
Original Date: 2002-04-05T23:47:58Z

---

Consolidating products to those we actually are maintaining.

**Comment by mbjones** [comment]
---

Original Redmine Comment
Author Name: **Redmine Admin** (Redmine Admin)
Original Date: 2013-03-27T21:13:28Z

---

Original Bugzilla ID was 152
